coralproject/talk
# [CORL-1187] Show media in the moderation view conversation modal

## What does this PR do?

Shows media in the moderation area's view conversation modal.

Moved the MediaContainer into the shared Admin components so that both the ModerateCard and the ViewConversation modal can use it.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- make sure you post some gifs/tweet/youtube to a story
- head to the admin moderation
- find the comment you posted
- click "View Conversation" on it
- see that the media shows in the modal conversation view
